src/features/script-load.js

```diff
@@ -5,18 +5,68 @@ import { hasDocument } from '../common.js';
 import { systemJSPrototype } from '../system-core.js';
 import { errMsg } from '../err-msg.js';
 
+if (hasDocument) {
+  window.addEventListener('error', function (evt) {
+    lastWindowErrorUrl = evt.filename;
+    lastWindowError = evt.error;
+  });
+  var baseOrigin = location.origin
+}
+
 systemJSPrototype.createScript = function (url) {
   var script = document.createElement('script');
   script.charset = 'utf-8';
   script.async = true;
-  script.crossOrigin = 'anonymous';
+  // Only add cross origin for actual cross origin
+  // this is because Safari triggers for all
+  // - https://bugs.webkit.org/show_bug.cgi?id=171566
+  if (!url.startsWith(baseOrigin + '/'))
+    script.crossOrigin = 'anonymous';
   script.src = url;
   return script;
 };
 
+// Auto imports -> script tags can be inlined directly for load phase
```

Comment:
This part seems to have been unintentionally carried over from https://github.com/systemjs/systemjs/pull/2223. Might be best to keep the two branches separate

Reply:
This code path is the auto import - I've merged it into the script load here.

Reply:
It is harder to review pull requests when they are not isolated to one feature, and the same code is present in multiple PRs.

Reply:
Ah I see, yes this was a rebasing issue! Thanks, fixed.